editor/project_export.cpp

```diff
@@ -1199,7 +1207,7 @@ ProjectExportDialog::ProjectExportDialog() {
 	updating = false;
 
 	get_cancel_button()->set_text(TTR("Close"));
-	get_ok_button()->set_text(TTR("Export PCK/Zip"));
+	get_ok_button()->set_text(TTR("Export PCK/ZIP..."));
 	export_button = add_button(TTR("Export Project"), !DisplayServer::get_singleton()->get_swap_cancel_ok(), "export");
```

Comment:
If the other two get `...` added, shouldn't "Export Project" get it too? It also spawns a dialog.